src/AboutDialog.cpp

```diff
@@ -51,22 +51,22 @@ hold information about one contributor to Audacity.
 #include "theme/AllThemeResources.h"
 #include "theme/Theme.h"
 
-#include "../images/SaucedacityLogoWithName.xpm"
+#include "../images/TenacityLogoWithName.xpm"
 
 #ifndef REV_TIME
 #define REV_TIME "unknown date and time"
 #endif
 
 #ifdef REV_LONG
-#define REV_IDENT wxString( "[[https://github.com/saucedacity/saucedacity/commit/" )+ REV_LONG + "|" + wxString( REV_LONG ).Left(6) + "]] of " +  REV_TIME 
+#define REV_IDENT wxString( "[[https://github.com/tenacity/tenacity/commit/" )+ REV_LONG + "|" + wxString( REV_LONG ).Left(6) + "]] of " +  REV_TIME 
```

Comment:
CAREFUL.

Reply:
Automation error...again...

Reply:
Yes, but I am not sure if this is going to work on Codeberg as-is.

Reply:
When we move to Codeberg, I'll cherry pick changes where git-describe is used.